degov.yml

```diff
@@ -0,0 +1,54 @@
+name: DeGov Development Test DAO
+logo: "/example/logo.svg"
+description: |
+  This is the development test DAO interface for DeGov users, allowing them to gain 
+  firsthand experience creating proposals and voting on them before deploying 
+  their own DeGov instance. The DAO is built on the Darwinia chain, an Ethereum-compatible chain.
+# The following fields in the links are optional and will be used to display basic information about the DAO
+links:
+  website: https://ringdao.com/
+  twitter: https://x.com/ringecosystem
+  discord: https://discord.com/invite/BhNbKWWfGV
+  telegram: https://t.me/ringecosystem
+  github: https://github.com/ringecosystem/degov
+  email: # It's ok to leave it empty
+
+walet:
+  walletConnectProjectId: 2719448e2ce94fdd269a3c8587123bcc
+
+chain:
+  # name: darwinia
```

Comment:
Please reorder this, make the must-have options before the nice-to-have options. In addition, please add some comments to explain which are must-have and which are nice-to-have.